packages/server/test/integration/cypress_spec.js

```diff
@@ -1159,6 +1159,8 @@ describe('lib/cypress', () => {
             close: sinon.stub().resolves(),
           }
 
+          sinon.stub(chromeBrowser, '_writeExtension').resolves()
```

Comment:
another victim of node.js's new fs strictness - can no longer write 'undefined' directly to a file